src/persist-client/src/rpc.rs

```diff
@@ -943,7 +943,7 @@ impl PersistGrpcPubSubServer {
     /// Starts the gRPC server. Consumes `self` and runs until the task is cancelled.
     pub async fn serve(self, listen_addr: SocketAddr) -> Result<(), anyhow::Error> {
         tonic::transport::Server::builder()
-            .add_service(ProtoPersistPubSubServer::new(self))
+            .add_service(ProtoPersistPubSubServer::new(self).max_decoding_message_size(usize::MAX))
```

Comment:
Should we add a comment here about the use of `usize::MAX`?

Reply:
I'll add a comment, the reason I didn't use `MAX_GRPC_MESSAGE_SIZE` is that this crate doesn't rely on the  `service/grpc` library in the same way that the other servers do.